common/src/main/java/tachyon/security/authorization/FsPermission.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security.authorization;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+
+/**
+ * A class for file/directory permissions.
+ */
+public class FsPermission {
+  //POSIX permission style
+  private FsAction mUseraction = null;
+  private FsAction mGroupaction = null;
+  private FsAction mOtheraction = null;
+
+  /**
+   * Construct by the given {@link FsAction}.
+   * @param u user action
+   * @param g group action
+   * @param o other action
+   */
+  public FsPermission(FsAction u, FsAction g, FsAction o) {
```

Comment:
We usually have the convention that the argument name is usually the same as the Type but with lower case for the first character. For example, `FsAction fsAction`. This will improve the code readability. Thanks.